alpha/veneer/README.md

```diff
@@ -0,0 +1,22 @@
+
+## Veneers 
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions.  FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/).  Unfortunately, FBCs can get complex, especially as the number of releases and dependencies scale.
+
+Enter the `veneer`, a simpler transitional step to generating an FBC.  The term is a bit of a misnomer, because a `veneer` isn’t pretending to be something else.
+
```

Comment:
Maybe a few words about how it's structured to give the reader a better idea?
For instance, a veneer is a custom process that uses an internal library to convert arbitrary input into a valid FBC.

Reply:
updated in docs PR